products/spring-framework.md

```diff
@@ -109,11 +117,18 @@ for more details about support roadmap.
 Extended support is available
 [from VMWare](https://tanzu.vmware.com/content/blog/vmware-spring-runtime-extended-support).
 
-## [Java Compatibility](https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Versions#jdk-version-range)
+## [Java/Jakarta Compatibility](https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Versions#jdk-version-range)
 
 {%- assign collapsedCycles = page.releases | collapse_cycles:"supportedJavaVersions"," - " %}
 {% include table.html
   labels="Release,Java (LTS)"
   fields="releaseCycle,supportedJavaVersions"
   types="string,string"
   rows=collapsedCycles %}
+
+{%- assign collapsedCycles = page.releases | collapse_cycles:"supportedJakartaVersions"," - " %}
```

Comment:
Would have prefered a single table, even if loosing collapsed cycles. WDYT ?

Reply:
I updated Java and Jakarte as custom columns, but on website it is to wide: https://deploy-preview-4110--endoflife-date.netlify.app/spring-framework
@marcwrobel  WDYT?

Reply:
Sorry I misled you, I meant a single table but not using a custom column. Will do the change.

Reply:
Done.